src/components/FilterSearch.tsx

```diff
@@ -95,6 +105,8 @@ export interface FilterSearchProps {
   onSelect?: (params: OnSelectParams) => void,
   /** A function which is called when the input element's value changes. Replaces the default behavior. */
   onDropdownInputChange?: (params: OnDropdownInputChangeProps) => void,
+  /** A function which is called when the input gains focus. In addition to the default behavior. */
```

Comment:
nit: can we update this to something like `A function which is called immediately after the input gains focus. It does not replace the default focus behavior.`

Reply:
Done